markdown/chapters/entombsday/interaction.md

```diff
@@ -0,0 +1,57 @@
+---
+authors: drynne
+order: 3
+title: Playing Around Interaction
+---
+
+## Table of Contents
+
+## Preamble
+
+In general, the easiest way to play around interaction is using discard and countermagic. However, because the core of the Tin Fins combo is Instant speed, there are a number of ways to play around different pieces interaction on the stack without those effects.
+
+## Leyline Effects
+
+:::row{variant=CENTERED}
+- Leyline of the Void
+- Rest in Peace
+- Dauthi Voidwalker
+:::
+
+These effects are the primary reason we play the Doomsday transformation in the sideboard. Nothing feels better than your opponent mulling to two or three for a Leyline of the Void and casting Dark Ritual into Doomsday on your turn.
+
+While Leyline of the Void and Rest in Peace are functionally blank cards against the Doomsday plan, Dauthi Voidwalker still poses a threat. Because Dauthi Voidwalker allows the opponent to cast cards exiled with it, you'll need to be careful about what cards you cast, discard and mill. Notably, this affects piles using :card[Consider] when the card being milled could be useful to the opponent: Force of Will, Daze, Flusterstorm, or even Emrakul, the Aeons Torn.
+
+## Cards that Empty the Graveyard
+
+:::row{variant=CENTERED}
+- Bojuka Bog
+- Endurance
+- Faerie Macabre
+:::
+
+Cards like Bojuka Bog, Endurance, and Faerie Macabre are some of the easiest to play around for Tin Fins. The general play pattern to beat these effects is to *bin* your Griselbrand, bait the interaction with a Shallow Grave, the simply cast a second copy of Entomb (or crack LED) to bin another Griselbrand after the effect resolves. 
+
+Note that Shallow Grave is required here, as it does not target a specific card.
+
+## Surgical Extraction
+
+:::row{variant=CENTERED}
+- Surgical Extraction
+- Extirpate
+:::
+
+While similar to the previously mentioned effects, Surgical Extraction requires the inverse play pattern. Since Surgical Extraction and Extirpate exile all copies of the chosen card, you can't allow it to resolve. To beat these effects, you will need two reanimation effects instead of two Entomb effects. 
+
+## Removal
+
+:::row{variant=CENTERED}
+- StP
+- Solitude
+- Karakas
+:::
+
+Tin Fins is able to play around removal on Griselbrand fairly easily. Most players will wait until you activate Griselbrand the first time before casting a removal spell. Because of this, you can usually activate Griselbrand a second time to find a piece of countermagic to answer the removal spell. 
+
+If you're unable to counter the removal, such as Karakas, it is still correct to draw as many cards as possible before it resolves. Generally, drawing 14 cards is enough to allow you to combo again in the same turn.
```

Comment:
Maybe talk about what cards you are looking to find from activations and how to know when to stop and let it happen?